lib/squake/pricing.rb

```diff
@@ -0,0 +1,56 @@
+# typed: strict
+# frozen_string_literal: true
+
+# https://docs-v2.squake.earth/group/endpoint-combined-calculation-pricing
```

Comment:
NIT: move above the `class` so IDEs don't show this comment for the module, but for the class on-hover.